README.rst

```diff
@@ -81,6 +81,14 @@ currently emitted:
   an instance of ``cls``, and ``__new__`` methods.
 * Y020: Quoted annotations should never be used in stubs.
 * Y021: Docstrings should not be included in stubs.
+* Y022: Imports linting: use typing-module aliases to stdlib objects as little
+  as possible (e.g. use ``builtins.list`` instead of ``typing.List``). However,
+  use ``typing.NamedTuple`` instead of ``collections.namedtuple``, as it allows
+  for more precise type inference. For typing-only objects such as ``ClassVar``
+  or ``NoReturn``, import objects from ``typing`` instead of
+  ``typing_extensions`` wherever possible.
```

Comment:
I feel like this should be split in three:
- one code for PEP 585 stuff (builtins instead of typing)
- one for NamedTuple
- one for using typing instead of typing-extensions where applicable

Reply:
It's annoying because from the perspective of writing the code, it's the same check. But I see what you mean, I'll revise it 🙂